src/test/header-manager.test.ts

```
@@ -0,0 +1,370 @@
+      headerManager = new HeaderManager();
+    });
+
+    it("devrait appeler destroy sur tous les sous-modules", () => {
+      const scrollHandler = headerManager.getScrollHandler();
+      const mobileMenu = headerManager.getMobileMenu();
+      const languageManager = headerManager.getLanguageManager();
+      const themeManager = headerManager.getThemeManager();
+
+      headerManager.destroy();
+
+      expect((scrollHandler as any).destroy).toHaveBeenCalledTimes(1);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)